README.md

```diff
@@ -16,7 +16,12 @@
 
 PlantUML Server is a web application to generate UML diagrams on-the-fly.
 
-[PlantUML is **not** affected by the log4j vulnerability.](https://github.com/plantuml/plantuml/issues/826)
+> [PlantUML is **not** affected by the log4j vulnerability.](https://github.com/plantuml/plantuml/issues/826)
+
+> **Breaking changes**:
+> PlantUML Server sets `PLANTUML_SECURITY_PROFILE` to `INTERNET` by default starting with version `v1.2023.9`.
+> You can change its behavior back to work like before if you set the environment variable `PLANTUML_SECURITY_PROFILE` to `LEGACY`.
+> But before you do that, please take a look to [PlantUMLs Security](https://plantuml.com/security) page.
```

Comment:
If the PR is delayed, we need to keep in mind to update the release note accordingly later.